libs/cluster/Server/ClusterManagerSlotState.cs

```diff
@@ -447,10 +447,14 @@ public void ResetSlotState(int slot)
         /// <param name="slots">Slot list</param>
         public void ResetSlotsState(HashSet<int> slots)
         {
-            foreach (var slot in slots)
+            while (true)
```

Comment:
Not familiar with this code, but is there a scenario where this doesn't break?

Reply:
I checked again to make sure, but it is a common pattern that we use elsewhere. Because we update config in parallel we need to retry until CAS succeeds. Maybe the confusion is on the name? I will prefix it with Try